fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -740,7 +740,7 @@ protected void checkForInsufficientStorageException(final Throwable rootThrowabl
     protected void addResourceHttpHeaders(final FedoraResource resource) {
         super.addResourceHttpHeaders(resource);
 
-        if (session.isBatchSession()) {
+        if (!transaction.isShortLived()) {
```

Comment:
this is fine for now, but i'm looking forward to this transformation going away :)